servicebox-app/src/test/java/io/halkyon/ClaimsEndpointTest.java

```diff
@@ -33,6 +33,18 @@ public void testAddFruit(){
                 .extract().body().jsonPath().getString("id");
     }
 
+    @Test
+    public void testAddClaimViaHtmxForm(){
+        // An htmx request will contain a HX-Request header and Content-Type: application/x-www-form-urlencoded
+        given()
+                .header("HX-Request", true)
+                .contentType(MediaType.APPLICATION_FORM_URLENCODED)
+                .body(CLAIM.apply("Oracle"))
```

Comment:
Should we create a body including name and serviceRequested fields ?

Reply:
yes, why not. In this code Oracle is the `name` (see [here](https://github.com/halkyonio/primaza-poc/pull/13/commits/931b2df9a009f0c6cf65490e941b82a99fd6eeed#diff-09d3f11861a602277f395756de607007a15b491793a187523cd4028e175a1611R19)) but I can add also a serviceRequested.